web/message_bar.css

```diff
@@ -0,0 +1,174 @@
+/* Copyright 2024 Mozilla Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.messageBar {
+	--closing-button-icon: url(images/messageBar_closingButton.svg);
```

Comment:
The formatting looks "off" all throughout this file, since it seems that you're using tabs (which we don't use anywhere in the code-base).

Running `gulp lint --fix` locally should help fix that, but I'd recommend using an editor that supports `.editorconfig`-files to help avoid that kind of issue.

Reply:
I've been using an extension that picks up `.editorconfig` but it was misbehaving for some reason and another formatter was overriding the settings, I'll look into it. In the meantime, thanks for the heads up I'll run `gulp lint --fix` before future commits.